windows/index.js

```diff
@@ -70,11 +67,11 @@ const start = () => {
       maximizable: true,
     });
 
-    positioner = new Positioner(player).move('bottomCenter')
+    const positioner = new Positioner(player).move('bottomCenter'); // eslint-disable-line no-unused-vars
```

Comment:
is this line doing anything? because positioner is not used anywhere else and I had to disable it with eslint

Reply:
It's responsible for the position of the player window before it's getting minimized. Technically it shouldn't be assigned to a variable. The 'new' already does what it's suppose to do so it's safe to remove the assignment. Good catch 🙏